lib/main.ts

```diff
@@ -137,16 +165,15 @@ export async function activate() {
 
   if (sneezeDataProvider) {
     // Test subscribing to a data provider
-    const unsubGreetings = await sneezeDataProvider.subscribeAchYou(
+    const unsubGreetings = await sneezeDataProvider.subscribeSneeze(
       "c897cd73-9100-4e6a-8a32-fe237f1e9928",
-      (timSneeze: Sneeze[] | User[]) =>
+      (timSneeze: Sneeze[]) =>
         logger.info(
           `Tim sneezed the ${
             (timSneeze as Sneeze[])[timSneeze.length - 1].sneezeId
```

Comment:
Again here you hopefully shouldn't need to do any type assertion for `timSneeze` (the `as Sneeze[]` part) as it is known to be `Sneeze[]` already

Reply:
Good point.

Reply:
Did you still have to type assert `timSneeze as Sneeze[]` here?